sickbeard/webserve.py

```diff
@@ -205,6 +205,9 @@ def render(self, *args, **kwargs):
             kwargs['title'] = '500'
             kwargs['header'] = 'Mako Error'
             kwargs['backtrace'] = RichTraceback()
+            for (filename, lineno, function, line) in kwargs['backtrace'].traceback:
+                logger.log(u'File %s, line %s, in %s' % (filename, lineno, function), logger.DEBUG)
+            logger.log(u'%s: %s' % (str(kwargs['backtrace'].error.__class__.__name__), kwargs['backtrace'].error))
```

Comment:
This came with another PR, i was working on. But would strongly advice to keep this in.